datahub-web-react/src/app/entity/shared/tabs/Dataset/Schema/SchemaTab.tsx

```diff
@@ -49,6 +55,60 @@ export const SchemaTab = ({ properties }: { properties?: any }) => {
     );
 
     const [showKeySchema, setShowKeySchema] = useState(false);
+    const [schemaViewMode, setSchemaViewMode] = useState(SchemaViewType.NORMAL);
+
+    const { data: getSchemaBlameVersionsData } = useGetSchemaBlameVersionsQuery({
+        skip: !datasetUrn,
+        variables: {
+            input: {
+                datasetUrn,
+                categories: [ChangeCategoryType.TechnicalSchema],
+            },
+        },
+    });
+    const latestVersion: string = getSchemaBlameVersionsData?.getSchemaBlame?.latestVersion?.semanticVersion || '';
+
+    const showSchemaBlame: boolean = schemaViewMode === SchemaViewType.BLAME;
+    const semanticVersionList: Array<SemanticVersionStruct> =
+        getSchemaBlameVersionsData?.getSchemaBlame?.semanticVersionList || [];
+    const version = useGetSemanticVersionFromUrlParams();
+    const selectedVersion = version || latestVersion;
+
+    const selectedSemanticVersionStruct = semanticVersionList.find(
+        (semanticVersion) => semanticVersion.semanticVersion === selectedVersion,
+    );
+    const selectedVersionStamp: string = selectedSemanticVersionStruct?.versionStamp || '';
+
+    let editMode = true;
```

Comment:
im wondering why we default to edit mode = true. i know this isn't your code.. but still what is going on here.

Reply:
(no action required)